.tekton/pull-request.yaml

```diff
@@ -182,6 +182,7 @@ spec:
           - build-bundles
         taskRef:
           name: e2e-test
+        timeout: "1h30m"
```

Comment:
if we want to fully utilize the timeout for generating attestation by chains controller of "1h30m" and also considering the other steps in the e2e-tests will take some time. Shall we increase it a little more something like "2h" ?
Also please add a comment with issue id for increasing the timeout.

Reply:
thanks! updated.